build/types-definitions.ts

```diff
@@ -1,19 +1,26 @@
+import process from 'process'
 import path from 'path'
 import fs from 'fs/promises'
 import * as vueCompiler from 'vue/compiler-sfc'
 import { Project } from 'ts-morph'
 import glob from 'fast-glob'
 import { bold } from 'chalk'
-
 import { errorAndExit, green, yellow } from './utils/log'
 import { buildOutput, epRoot, pkgRoot, projRoot } from './utils/paths'
+import typeSafe from './type-safe.json'
 
 import { excludeFiles, pathRewriter } from './utils/pkg'
 import type { SourceFile } from 'ts-morph'
 
 const TSCONFIG_PATH = path.resolve(projRoot, 'tsconfig.json')
 const outDir = path.resolve(buildOutput, 'types')
 
+const typeSafePaths = typeSafe.map((_path) => {
+  let safePath = path.resolve(projRoot, _path)
+  if (_path.endsWith('/')) safePath += path.sep
+  return safePath
+})
+
```

Comment:
Could you please add a documentation here elaborating what is this all about?

Reply:
Hmmmm? Is it really necessary? Currently the build system is maintained by me alone.

Reply:
And this is only an alternative until the TS error is fully resolved.

Reply:
No offense, that is why we need to document it.

Reply:
Done.